### PR TITLE
Overwrite server.conf settings only for provided env vars

### LINKED ISSUE #63: [Feature] Support setting changes during runtime
**Info**
It is now possible to change settings during runtime

**Problem**
Everytime the server gets started, these settings will be overwritten by the environment variables

**Solution**
Do not overwrite the settings with environment variables.

You probably don't want to use them at all, since they just create maintenance work to keep up with the server setting changes.
The server keeps its server config up-to-date by removing old and adding new settings.
This however will not be reflected in the available env variables unless this repo gets updated as well

### PR DESCRIPTION
**Pull request by schroda**
Currently on every start the whole server.conf file gets replaced which causes any changed settings during runtime to get lost

closes #63

### CODE REVIEW

File: scripts/create_server_conf.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# exit early in case the file already exists
+if [ -f /home/suwayomi/.local/share/Tachidesk/server.conf ]; then
+    exit 0
+fi
+
+mkdir -p /home/suwayomi/.local/share/Tachidesk
+
+# extract the server reference config from the jar
+unzip -q /home/suwayomi/startup/tachidesk_latest.jar -d /home/suwayomi/startup/tachidesk_latest
```
Comment (Syer10) [inline scripts/create_server_conf.sh]: I think we should only extract the server.conf

**Review by Syer10 — CHANGES_REQUESTED** [review]

File: scripts/create_server_conf.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# exit early in case the file already exists
+if [ -f /home/suwayomi/.local/share/Tachidesk/server.conf ]; then
```
Comment (chancez) [inline scripts/create_server_conf.sh:4]: This means the file will not get updated even if you change env-vars, which feels sort of confusing.

Comment (Syer10) [reply scripts/create_server_conf.sh:4]: It will, this script just creates the config for the first time, its not used after that

**Comment by chancez** [comment]
I think it would still be useful to have a `GENERATE_CONFIG` option in case we want to manage the config ourselves. I was planning on storing my config in git and this will certainly cause the file to have some diffs if it’s going to always call sed.

**Pull request merged**